src/AaveV3StrategicAssets_20230606/StrategicAssetsManager.sol

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.19;
+
+import {IERC20} from 'solidity-utils/contracts/oz-common/interfaces/IERC20.sol';
+import {Initializable} from 'solidity-utils/contracts/transparent-proxy/Initializable.sol';
+import {SafeERC20} from 'solidity-utils/contracts/oz-common/SafeERC20.sol';
+
+import {LSDLiquidityGaugeManager} from './LSDLiquidityGaugeManager.sol';
+import {VeTokenManager} from './VeTokenManager.sol';
+
+contract StrategicAssetsManager is Initializable, LSDLiquidityGaugeManager, VeTokenManager {
```

Comment:
There are missing interfaces, I'd create and inherit from those interfaces, while also inheriting NatSpec to make things clearer

Reply:
Not sure the value of this one? Is it so somebody can grab the interface from here when doing a payload for example?

Reply:
It's just more readable imo, I'm actually indifferent to having the NatSpec be inherited from the interface, but I do like being able to look at a contract's interface and understand what it does without having to see the body of every function for example. It also helps to have interfaces for integrators as you mention.

Overall not a huge deal as there's no real implication on-chain, but still a nice to have.